packages/core/admin/ee/server/services/review-workflows/review-workflows.js

```diff
@@ -24,31 +37,111 @@ function buildStagesConnectArray(stages) {
+    await enableReviewWorkflow({ strapi })({ contentTypes: strapi.contentTypes });
+  }
+}
+
+const setStageAttribute = set(`attributes.${ENTITY_STAGE_ATTRIBUTE}`, {
+  writable: true,
+  private: false,
+  configurable: false,
+  visible: false,
+  type: 'relation',
+  relation: 'morphOne',
+  target: 'admin::workflow-stage',
+  morphBy: 'related',
+});
+
+function extendReviewWorkflowContentTypes({ strapi }) {
+  const extendContentType = (contentTypeUID) => {
+    strapi.container.get('content-types').extend(contentTypeUID, setStageAttribute);
+  };
+  pipe([
+    getContentTypeUIDsWithActivatedReviewWorkflows,
+    // Iterate over UIDs to extend the content-type
+    forEach(extendContentType),
+  ])(strapi.contentTypes);
+}
+
+/**
+ * Enables the review workflow for the given content types.
+ * @param {Object} strapi - Strapi instance
+ */
+function enableReviewWorkflow({ strapi }) {
+  /**
+   * @param {Array<string>} contentTypes - Content type UIDs to enable the review workflow for.
+   * @returns {Promise<void>} - Promise that resolves when the review workflow is enabled.
+   */
+  return async ({ contentTypes }) => {
+    // TODO To be refactored when multiple workflows are added
+    const defaultWorkflow = await strapi
+      .query(WORKFLOW_MODEL_UID)
+      .findOne({ populate: ['stages'] });
+
+    // This is possible if this is the first start of EE, there won't be any workflow in DB before bootstrap
+    if (!defaultWorkflow) {
+      return;
+    }
+    const firstStage = defaultWorkflow.stages[0];
+    const up = async (contentTypeUID) => {
+      const contentTypeMetadata = strapi.db.metadata.get(contentTypeUID);
+      const { target, morphBy } = contentTypeMetadata.attributes[ENTITY_STAGE_ATTRIBUTE];
+      const { joinTable } = strapi.db.metadata.get(target).attributes[morphBy];
+      const { idColumn, typeColumn } = joinTable.morphColumn;
+
+      // Execute a raw SQL query to insert records into the join table mapping the specified content type with the first stage of the default workflow.
+      // Only entities that do not have a record in the join table yet are selected.
+      await strapi.db.connection
+        .raw(`INSERT INTO ${joinTable.name} (${idColumn.name}, field, "order", ${joinTable.joinColumn.name}, ${typeColumn.name})
+                SELECT
+                    entity.id as ${idColumn.name},
+                    '${ENTITY_STAGE_ATTRIBUTE}' as field,
+                    1 as "order",
+                    ${firstStage.id} as ${joinTable.joinColumn.name},
+                    '${contentTypeUID}' as ${typeColumn.name}
+                FROM ${contentTypeMetadata.tableName} entity
+                LEFT JOIN ${joinTable.name} jointable
+                ON  entity.id = jointable.${idColumn.name}
+                AND jointable.${typeColumn.name} = '${contentTypeUID}'
+                WHERE jointable.${idColumn.name} IS NULL`);
+    };
```

Comment:
For future PRs :) I would do this in a knex query, also, adding API tests  would ensure this works for all dbs.

⚠️  I have not tried this query, but it should be something like this.
```js
const con = strapi.db.connection;
const selectStatement = con
  .select(
    `entity.id as ${idColumn.name}`,
    `${ENTITY_STAGE_ATTRIBUTE} as field`,
    con.raw(`?? as order`, [1]), // TODO : Add workflow id
    `${firstStage.id} as ${joinTable.joinColumn.name}`,
    `${contentTypeUID} as ${typeColumn.name}`
  )
  .leftJoin(`${joinTable.name} jointable`, function() {
      this
        .on('entity.id', '=', `jointable.${idColumn.name}`)
        .andOn(`jointable.${typeColumn.name}`, '=',  `${contentTypeUID}`)
  })
  .where(`jointable.${idColumn.name}`, null)
  .from(joinTable.name)
  .toSQL();

// Insert the clone relations
await this.createQueryBuilder(joinTable.name)
  .insert(
    con.raw(
      `(${columnsToInsert.join(',')})  ${selectStatement.sql}`,
      selectStatement.bindings
    )
  )
  .execute();
```